src/models/ontology_mapping_engine.py

```diff
@@ -0,0 +1,128 @@
+from src.models import ontology_mapper_st as oms
+from src.models import ontology_mapper_lm as oml
+from OmicsMLRepoHarmonizer.src.models import ontology_mapper_rag as ollm
+import src.CustomLogger.custom_logger
+import pandas as pd
+import numpy as np
+from thefuzz import fuzz
+
+logger = src.CustomLogger.custom_logger.CustomLogger()
+
+
+
+class OntoMapEngine():
+    def __init__(self, 
+                 method:str, 
+                 query:list[str], 
+                 corpus:list[str],
+                 cura_map:dict, 
```

Comment:
FYI, for production, `cura_map` shouldn't be required because they will not be available for new queries.

Reply:
Yes I agree - I will make the changes accordingly

Reply:
I've added the test_or_prod parameter to get results in testing or production. Additionally I've committed a demo notebook to show results for testing or production environment in v0.2.4 updates